### DISCUSSION TITLE
Subset out of bounds

### DISCUSSION THREAD

**Discussion by SGKhir** [ImmuneDynamics/Spectre#49] [Help]
Hi,

I am new to using Spectre and after running a demo file, I am not trying to run my file. After selecting the cols, when i run Flowsom I get this error:

Preparing data
Error: Subset out of bounds

I couldn't find any solution to this. Could someone please help? Thanks

**Comment by tomashhurst** [comment]
Hey @SSKhir, it sounds like the code to select the cols might be a little messed up -- could you copy and paste the code (and the results)?
